tasks/Debian-tarball.yml

```diff
@@ -0,0 +1,12 @@
+---
+- include: tarball.yml
+
+- include: upstart.yml
+  tags: deploy
+
+- include: systemd.yml
+  tags: deploy
```

Comment:
OK, then I suggest to make it configurable for the user, preferably by introducing 1 new variable:
```
# when true systemd will be used (and *not* upstart), when 'false' the other way
zookeeper_debian_systemd_enabled: false (by default)
```

Even better: Set the default setting depending on the ubuntu version (I propose to set it to true only for Ubuntu 16.04, where systemd seems to be the new default init system, and replace upstart)

Alternative we could use 2 separate variables, which would allow to enable/disable both systemd/upstart independently  (could also re-use for redhat, combining those flags with the 'systemd/upstart' exists check ?!)